shm/Cargo.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <string>
+
+class Cargo {
```

Comment:
Please document class in header. You should write description of class and interface. Sometimes when private functions are compilated you can also document it.

```C++
// Class responsible for manage Currency in game. 
class Money {
public:
    // Function will rise price for every item in game.
    void makeInflation();

private:
}|;
```